cmd/manager-hub/main.go

```diff
@@ -108,8 +108,8 @@ func main() {
 	metricsAPI.Register()
 
 	registryAPI := registry.NewRegistry()
-	buildSignPodAPI := buildsignpod.NewBuildSignPodManager(client)
 	buildSignCombiner := module.NewCombiner()
+	buildSignPodAPI := buildsignpod.NewBuildSignPodManager(client, buildSignCombiner, scheme)
 
 	buildAPI := buildpod.NewBuildManager(
```

Comment:
why do we still new buildAPI (and Sign API). Did we forgot to remove it?

Reply:
> Did we forgot to remove it?

Yes.